srcpkgs/j/template

```diff
@@ -29,7 +29,7 @@ aarch64*) _jpl="raspberry";;
 esac
 
 case "${XBPS_TARGET_MACHINE}" in
-*-musl)   makedepends+=" musl-fts-devel";;
+*-musl) makedepends="${makedepends} musl-fts-devel";;
```

Comment:
use `makedepends+=" musl-fts-devel"` instead

Reply:
Oh, right. I meant to ask about that first. It seems like `+=` used by convention, but shellcheck complains that `+=` is non-POSIX. Will change, but that surprised me.

Reply:
xbps-src needs bash because POSIX shell does not have some features we use. This is one of them.